UKHomeOffice/nmsw-ui
# NMSW-470 Refactor declarationId - file uploads & error pages

# Ticket

NMSW-470

----

## AC

see https://github.com/UKHomeOffice/nmsw-ui/pull/269

----

## To test
- create a voyage
- add a bad general declaration file
- click check for errors
- > you should see the error page
- go back
- add a good general declaration file
- click check for errors
- > you should see the success page
- click save and continue
- click crew details
- add a bad crew page
- click check for errors
- > you should see the crew error page
- go back
- add a good crew page
- click check for errors
- > you should see the success page


----

## Notes
